e2e/tests/interchain_accounts/intertx_test.go

```diff
@@ -29,6 +31,18 @@ type InterTxTestSuite struct {
 	testsuite.E2ETestSuite
 }
 
+// getICAVersion returns the version which should be used in the MsgRegisterAccount broadcast from the
+// controller chain.
+func getICAVersion(chainAVersion, chainBVersion string) string {
+	chainBIsGreaterThanOrEqualToChainA := semver.Compare(chainAVersion, chainBVersion) <= 0
```

Comment:
It looks like we have a function that does the GTE check [here](https://github.com/cosmos/ibc-go/blob/143e5d85e9d2fc5fc2fc53adc42127a439ee2b79/e2e/semverutil/semver.go#L52-L55)

Maybe we can export it and rename it to `GTE` so we can do `semverutil.GTE` directly here.

WDYT?

Reply:
Thanks for the suggestion, @chatton. I have implemented the proposed change.